server/chat-plugins/randombattles/index.ts

```diff
@@ -846,6 +850,9 @@ export const commands: Chat.ChatCommands = {
 		}
 
 		if (!movesets.length) {
+			if (isInexact) {
+				return this.errorReply(`Error: No Pok\u00e9mon named '${args[0].trim()}' was found. (Check your spelling?)`);
+			}
```

Comment:
Still not sure why this is necessary. If we couldn't find a species match, we should only need to error out for that once.

Reply:
This isn't checking if we find a species match.  This is checking if the species has a moveset in randbats.  Some mons, such as NFEs and babies, don't have sets in randbats.  That's what this check is doing, and it's only in one spot.

Reply:
I don't follow why the `isInexact` check is necessary then? We should be erroring `species does not exist` only once, and `moveset data doesn't exist` is handled _below_ this chec,

Reply:
I also think that if we did fuzzy searching and didn't find a moveset, then we just say no pokemon with the original term was found.  E.g. if they misspell but meant toadscruel and the fuzzy search matches toedscool instead, we shouldn't say that toedscool has no random battles data, we should say the misspelled Pokemon couldn't be found.

Since we're only doing this in one spot now, I put it here.

Reply:
If we didn't find a moveset, it should say that. Saying something else is counterproductive.

Reply:
I guess my issue is that I'm not communicating to the user that fuzzy searching was performed at all. So if we don't notify the user that fuzzy searching is done, and fuzzy searching selects a Pokemon without moves, then we say no moves found.

The toedscool/toedscruel example I gave illustrates my hesitation with leaving the error messages as is.

I'll add that to the response box when fuzzy searching was done (just like the DT command).

Reply:
Yeah, would prefer a dt-esque notice instead of this particular error message. That'll make it clear, I don't think this error message currently does otherwise.

Reply:
Agreed, l'll figure out how to hook into the response box.  Thanks for the suggestions

Reply:
```ts
	for (const [i, newTarget] of newTargets.entries()) {
			if (newTarget.isInexact && !i) {
				buffer = `No Pok\u00e9mon, item, move, ability or nature named '${target}' was found${Dex.gen > dex.gen ? ` in Gen ${dex.gen}` : ""}. Showing the data of '${newTargets[0].name}' instead.\n`;
			}``` 
This is what `/dt` does.